### PR TITLE
Refactor/fetch homepage featured deals from deals service

### PR DESCRIPTION
**Pull request by giladt**
## What was done
Removed old Mock from `home.ts` and implemented it in the current ceramic mock.
Fixed `dealSummary` to implement changes in the mock
Fixed the `timeLeft` statuses to have the correct colors for each status.
Fixed `DealTokenSwap` to implement status in the deal object.
Added `DealStatus` enum in `IDealTypes.ts`.
Fixed navigation route to the correct deal dashboard route.

### CODE REVIEW

File: src/entities/DealTokenSwap.ts [hunk 1]
```diff
@@ -84,6 +84,7 @@ export class DealTokenSwap implements IDeal {
       this.registrationData = await this.dataSourceDeals.get<IDealRegistrationTokenSwap>(this.rootData.registration);
       const discussionsMap = await this.dataSourceDeals.get<Record<string, string> | undefined>(this.rootData.discussions);
       this.clauseDiscussions = new Map(Object.entries(discussionsMap ?? {}));
+      this.status = this.rootData.status;
```
Comment (dkent600) [inline src/entities/DealTokenSwap.ts:87]: fake "status" should not be coming from Ceramic.  I think we need this coming from elsewhere for Deal entity stuff.

File: src/resources/elements/timeLeft/timeLeft.scss [hunk 2]
```diff
@@ -7,18 +7,29 @@ $comingUpColor: $Orange;
 $comingUpBackground: #3e3038;
 $farOffColor: #cecece;
 $farOffBackground: #30343e;
-$liveColor: $Green;
+
+$completedColor: #18bab0;
```
Comment (dkent600) [inline src/resources/elements/timeLeft/timeLeft.scss:11]: TO DO: These need to come from the color palette

File: src/entities/DealTokenSwap.ts (hunk 1 above)
Comment (dkent600) [reply src/entities/DealTokenSwap.ts:87]: Ideally we are deriving status by whatever logic it is supposed to be derived from

Comment (dkent600) [reply src/entities/DealTokenSwap.ts:87]: How about for now we just have each Deal return a random value for status.

Comment (giladt) [reply src/entities/DealTokenSwap.ts]: I've removed the ceramic implementation and added a (temporary) random status distributer in the DealTokenSwap entity.

File: src/resources/elements/timeLeft/timeLeft.scss (hunk 2 above)
Comment (giladt) [reply src/resources/elements/timeLeft/timeLeft.scss]: Moved the variables to the design system colors file. This still might change once @v0ort will update the palette in the Figma file.

**Review by dkent600 — APPROVED** [review]

**Pull request merged**